acceptance/tests/resource/user/should_allow_managed_macos_users_to_login.rb

```diff
@@ -35,7 +35,7 @@
       on(agent, "dscl /Local/Default -authonly testuser helloworld", :acceptable_exit_codes => 0)
     end
 
-    unless agent['platform'] =~ /osx-11/
+    unless agent['platform'] =~ /^osx-1[1-9]/
       skip_test "AuthenticationAuthority field fix test is not valid for macOS older than Big Sur (11.0)"
```

Comment:
I was confused by the message passed to `skip_test`. It was originally added in https://github.com/puppetlabs/puppet/commit/de14d588679b29394ea37e8c55ac9bd071b51b83#diff-36276fb0c0144bd182aaae5581b140b1271b2eef0dbd6dc5743e9bd818d90659R38-R40 but I think Luchi meant to say "the test is not valid for macOS **before** Big Sur (11.0)" So with your PR we now correctly run the steps that come later on `osx-11 - osx-19`